finalPrep/modZeroHero.js

```diff
@@ -56,29 +110,72 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 //   - luckyNumbers (array)
 //   - address (object with following key/values: number , street , state, zip)
 
+var scaryMonster = {
+    name: "Stay Puft Marshmallow Man",
+    smell: "Mildly sweet",
+    weight: 7500,
+    citiesDestroyed: ["New York", "Chicago"],
+    luckyNumbers: [1, 7, 13]
+};
+    var address = {
+      number: "1984",
```

Comment:
How would you make sure this data relates back to your scary monster? We don't need a separate declaration here, but address can be another key within scaryMonster that has a value of an object.